Source/Dafny/Compilers/Compiler-js.cs

```diff
@@ -2007,38 +2007,46 @@ protected class ClassWriter : IClassWriter {
           }
 
         case BinaryExpr.ResolvedOpcode.Lt:
-          if (e0.Type.IsNumericBased() && AsNativeType(e0.Type) == null) {
+          if (AsNativeType(e0.Type) != null) {
+            opString = "<";
+          } else if (e0.Type.IsNumericBased() || e0.Type.IsBitVectorType) {
             callString = "isLessThan";
           } else {
-            opString = "<";
+            Contract.Assert(false);
```

Comment:
AFAICT we always `throw new cce.UnreachableException();` after such a contract assertion. That ensures control flow doesn't just continue in release builds.